set_env.bat

```diff
@@ -0,0 +1,29 @@
+@echo off
+REM
+REM Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+REM or more contributor license agreements. Licensed under the Elastic License
+REM 2.0 and the following additional limitation. Functionality enabled by the
+REM files subject to the Elastic License 2.0 may only be used in production when
+REM invoked by an Elasticsearch process with a license key installed that permits
+REM use of machine learning features. You may not use this file except in
+REM compliance with the Elastic License 2.0 and the foregoing additional
+REM limitation.
+REM
+
+REM Set up a build environment, to ensure repeatable builds
+
+REM Initialize the Visual Studio command prompt environment variables
+cmd /c "C:\Program Files (x86)\Microsoft Visual Studio\2019\Professional\VC\Auxiliary\Build\vcvarsall.bat" x86_amd64
+
+REM Set %CPP_SRC_HOME% to be an absolute path to this script's location, as
+REM different builds will come from different repositories and go to different
+REM staging areas
+SET MY_DIR=%~dp0

```

Comment:
`MY_DIR` isn't needed here.  In `set_env.sh` it's used to simplify away symlinks when setting `CPP_SRC_HOME`, but used like it is here it's pointless.